scripts/example.ts

```diff
@@ -255,6 +261,12 @@ async function main() {
     const rendererStoriesDir = path.resolve(codeDir, `./renderers/${renderer}/src/stories`);
     await copyStories(rendererStoriesDir, path.join(storiesDir, renderer), { isTS });
 
+    // TODO: why can I not just use `@storybook/react/test-components`? @ndelangen?
```

Comment:
@ndelangen any idea why this wouldn't work? It seems like (according to `package.json:exports`) it points to the same place.